docs/user_guide/3_getting_started.md

```diff
@@ -178,11 +178,194 @@ When running the one-liner, along with the icon that starts the Grasper, you wil
+
+The following table has some more information on what each of these fields mean.
+
+```eval_rst
+============================   ==============================================================================================
+Item                           Description
+============================   ==============================================================================================
+Full Name
+Component
+Hardware ID
+Level
+Message                        Any error or status messages
+Motor ID                       This is the motor number. Range [0..19]
+Motor ID in message            For debugging only
+Strain Gauge Left / Right      These are the ADC readings from the two gauges
+Executed Effort
+Motor Flags                    See motor flags table below
+Measured current               Current flowing through the motor (Amps)
+Measured Voltage               The motor power supply voltage. Not the voltage at the motor
+Temperature                    The temperature measured near the motor. The actual motor winding temperature will be higher than this. (ºC)
+Number of CAN messages         Received messages should be twice the transmitted messages
+Force control P, I, D terms    These are the PID terms from inside the motor's torque controller. They may be useful for debugging if plotted.
+Force control F, P, I, D,
+Imax, Deadband, Sign           These are the FPID gain settings used by the motor's torque controller. They can be changed using the controller tuner.
+Last Measured Effort           Difference between the two gauge readings (Torque)
+Last Commanded Effort          Torque requested by the host-side control algorithms
+Encoder Position               The angle of the joint in radians (ROS always calls this Encoder position, even if the robot uses Hall effect sensors)
+Firmware svn revision          xxxx: The latest version of the firmware available at build time
+                               xxxx: The version of the firmware in the motor MCU
+                               False: There are no un-checked-in modifications to this firmware. This should never be true.
+============================   ==============================================================================================
+```
+
+### Controller tuner
+It is possible to adjust the settings for any of the Position or Force (Motor) controllers.
+	**Plugins → Shadow Robot → Basic → Controller Tuner**
+#### Position controller
+  ![adjust_position_controller](../img/adjust_position_controller.png)
+
+
+
+  Here you can select a finger, thumb or wrist joints, and adjust the different position control parameters. Click ```Set Selected``` to send the new values to the motors and make them take effect.
+
+* **“P”, “I” & “D” terms:**  Gains parameter of the position PID controller. By default, Shadow tunes the paramenters using P or PD combinations. The user can add “I” gains in the control if he considers necessary.
```

Comment:
"Gains parameter of the position PID controller." This sentence sounds weird to me. Could you propose a better way of defining this?